tests/sentry/api/endpoints/test_organization_integration_details.py

```diff
@@ -2,43 +2,67 @@
 
 import six
 
-from sentry.models import Integration, OrganizationIntegration
+from sentry.models import Integration, OrganizationIntegration, ProjectIntegration
 from sentry.testutils import APITestCase
 
 
 class OrganizationIntegrationDetailsTest(APITestCase):
-    def test_simple(self):
+    def setUp(self):
+        super(OrganizationIntegrationDetailsTest, self).setUp()
+
         self.login_as(user=self.user)
-        org = self.create_organization(owner=self.user, name='baz')
-        integration = Integration.objects.create(
+        self.org = self.create_organization(owner=self.user, name='baz')
+        self.integration = Integration.objects.create(
             provider='example',
             name='Example',
         )
-        integration.add_organization(org.id)
-        path = '/api/0/organizations/{}/integrations/{}/'.format(org.slug, integration.id)
+        self.integration.add_organization(self.org.id, config={'setting': 'value'})
 
-        response = self.client.get(path, format='json')
+        self.path = '/api/0/organizations/{}/integrations/{}/'.format(self.org.slug, self.integration.id)
+
+    def test_simple(self):
```

Comment:
please use a verbose test name.

Reply:
This fits naming convention for all of our other API endpoints.